spartan_parallel/examples/interface.rs

```diff
@@ -5,9 +5,8 @@ use std::io::{BufRead, Read};
 use std::{default, env};
 use std::{fs::File, io::BufReader};
 
-use libspartan::{
-  instance::Instance, InputsAssignment, MemsAssignment, SNARKGens, VarsAssignment, SNARK,
-};
+use libspartan::scalar::{Scalar, SpartanExtensionField};
```

Comment:
It makes more sense to import `ScalarExt2` instead of `Scalar` here as `Scalar` does not provide enough soundness for us.

Reply:
Changed to `ScalarExt2` as requested